extensions/sql-database-projects/src/models/project.ts

```diff
@@ -118,7 +120,10 @@ export class Project implements ISqlProject {
 	}
 
 	public get sqlProjStyleName(): string {
-		return this.sqlProjStyle === ProjectType.SdkStyle ? 'SdkStyle' : 'LegacyStyle';
+		if (utils.getAzdataApi()) {
```

Comment:
Can you make `'SdkStyle'` and `'LegacyStyle'` constants (maybe in sqldbproj.d.ts so that other extensions can use them to check the value)?

Reply:
Somehow making them constants in sqldbproj.d.ts was creating problems with extension activation. So for now, keeping the strings as-is.